src/main/java/org/folio/bulkops/processor/AbstractDataProcessor.java

```diff
@@ -21,40 +21,40 @@
 
   @Override
   public UpdatedEntityHolder process(String identifier, T entity, BulkOperationRuleCollection rules) {
-    var hasProcessingError = false;
     var holder = UpdatedEntityHolder.builder().build();
     var updated = clone(entity);
-
+    var countActions = 0;
+    var countErrors = 0;
     for (BulkOperationRule rule : rules.getBulkOperationRules()) {
       var details = rule.getRuleDetails();
       var option = details.getOption();
       for (Action action : details.getActions()) {
+        countActions++;
```

Comment:
countActions is unuseful here because we have totalRecords in the BulkOperationRuleCollection